zstacklib/zstacklib/utils/qga.py

```
@@ -190,6 +190,47 @@ def guest_exec_status(self, pid):
+            {"path": path, "arg": args, "capture-output": output})
+        if ret and "pid" in ret:
+            pid = ret["pid"]
+        else:
+            raise Exception('qga exec program {} failed for vm {}'.format(path, self.vm_uuid))
+
+        if not output:
+            logger.warn("run qga program: {} failed, no output".format(path))
+            return 0, None
+
+        ret = None
+        for i in range(retry):
```

Ruff (0.15.21)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)